client/src/main/scala/org/apache/celeborn/client/commit/ReducePartitionCommitHandler.scala

```diff
@@ -291,7 +307,12 @@ class ReducePartitionCommitHandler(
               val returnedMsg = GetReducerFileGroupResponse(
                 StatusCode.SUCCESS,
                 reducerFileGroupsMap.getOrDefault(shuffleId, JavaUtils.newConcurrentHashMap()),
-                getMapperAttempts(shuffleId))
+                getMapperAttempts(shuffleId),
+                pushFailedBatches =
+                  shufflePushFailedBatches.getOrDefault(
+                    shuffleId,
+                    JavaUtils.newConcurrentHashMap()).values().asScala.flatMap(x =>
+                    x.asScala.toSet[PushFailedBatch]).toSet.asJava)
```

Comment:
Will this not work ?

```suggestion
                    JavaUtils.newConcurrentHashMap()).values().asScala.flatMap(_.asScala).toSet.asJava)
```

Reply:
fixed, thanks